app/src/common/css/variables/colors.scss

```diff
@@ -30,6 +30,7 @@ $COLOR--light-yellow: #f5f9ec;
 $COLOR--orangish: #fb7848;
 $COLOR--tealish: #39c2d7;
 $COLOR--tealish-hover: #d7f3f7;
+$COLOR--tealish-light: #d7f2f7;
```

Comment:
Do not add any color variables without UX-Designer approve. The added color the same as $COLOR--tealish-hover, so there is no need to add new one.